### PR TITLE
Changed description of the responsibility of EntityIdParser

### PR DESCRIPTION
**Pull request by manicki**
* The fact that `EntityIdParser` is an interface does not seem relevant on this level of detail
* Also avoids "tautological" description that a parser parses

### CODE REVIEW

**Comment by manicki** [comment]
probably "transforms" would be a better word than "translates"

File: systems/WikibaseRepo/05-Building_Block_View.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ TBA
...
 | -------------------------------------- | ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | ModifyEntity                           | base class                    | Base class for API modules modifying a single entity                                                                                                                                                         |
-| EntityIdParser                         | Handling user input           | Interface for objects that can parse strings into EntityIds                                                                                                                                                  |
+| EntityIdParser                         | Handling user input           | Translates strings into EntityId objects                                                                                                                                                  |
```
Comment (tzhelyazkova) [inline systems/WikibaseRepo/05-Building_Block_View.md:42]: "Transforms" indeed sounds more correct  than "translates".

**Review by tzhelyazkova — COMMENTED** [review]
I don't think the original author meant to emphasize that this is an interface, but I do see your point.

File: systems/WikibaseRepo/05-Building_Block_View.md (hunk 1 above)
Comment (addshore) [reply systems/WikibaseRepo/05-Building_Block_View.md:42]: +1

**Review by addshore — APPROVED** [review]

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**